packages/grafana-ui/src/components/Stack/Stack.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { ThemeSpacingTokens } from '@grafana/data';
+
+import { Direction, Flex } from '../Flex/Flex';
+
+interface StackProps {
+  direction?: Direction;
+  gap?: ThemeSpacingTokens;
+}
+
+const Stack = ({ gap = 1, direction = 'column', children }: React.PropsWithChildren<StackProps>) => {
+  return (
+    <Flex gap={gap} direction={direction}>
+      {children}
+    </Flex>
+  );
+};
+
+Stack.displayName = 'Stack';
+export { Stack };
```

Comment:
Any particular reason why we export it like this instead of just exporting directly where we create it?

Reply:
Ah, sorry, this was since I used the forwardRef implementation. I don't think we need a ref for this one (it is probably more fitting for the Box component). I'll change it.